media/tabs/tab-debugger.html

```
@@ -84,6 +84,52 @@ <h3>Advanced Features</h3>
+                        <strong>Hit Conditional Breakpoints:</strong> Right-click a breakpoint, choose
+                        <em>Edit Hit Count…</em>, and enter a hit-count expression. The breakpoint only
+                        pauses when the condition is met.
+                        <table>
+                            <tr><th>Expression</th><th>Meaning</th><th>Example use</th></tr>
+                            <tr>
+                                <td><span class="code-inline">5</span> or <span class="code-inline">= 5</span></td>
+                                <td>Pause on the 5th hit exactly</td>
+                                <td>Skip the first 4 ticks and inspect only the 5th</td>
+                            </tr>
+                            <tr>
+                                <td><span class="code-inline">> 10</span></td>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)